src/style.css

```
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg) no-repeat, #fff;
```

I meant you should add background-position to common class .stars__star and remove it from other selectors

```suggestion
  background: url(./images/star.svg) no-repeat, #fff;
  background-position: center;
```